xbmc/interfaces/legacy/Control.h

```diff
@@ -1473,15 +1473,16 @@ namespace XBMCAddon
     /**
      * ControlSlider class.
      * 
-     * ControlSlider(x, y, width, height[, textureback, texture, texturefocus])
+     * ControlSlider(x, y, width, height[, textureback, texture, texturefocus, orientation])
      * 
      * x              : integer - x coordinate of control.\n
      * y              : integer - y coordinate of control.\n
      * width          : integer - width of control.\n
      * height         : integer - height of control.\n
      * textureback    : [opt] string - image filename.\n
      * texture        : [opt] string - image filename.\n
-     * texturefocus   : [opt] string - image filename.n"
+     * texturefocus   : [opt] string - image filename.\n
+     * orientation    : [opt] integer - orientation of slider (VERTICAL / HORIZONTAL).n"
```

Comment:
does VERTICAL and HORIZONTAL exist in the python api? if not, it should be mapped, or docs changed to '0 or 1'